assignment0

```diff
@@ -0,0 +1,23 @@
+## Assignment 0: Hello Google Sheets!
+
+Graded Assignments will count towards your final grade in the course. Each assignment is graded pass-fail. While Assignment 0 is more of a tutorial for the learners to get you used to submitting Google Sheets for grading, they need a passing grade on each assignment to complete the course. Each assignment builds on concepts introduced over multiple lessons, so we would them to begin reviewing the assignment and working on the different tasks as soon as the assignment becomes available in Canvas.
+
+Their first assignment is mostly an exercise in following instructions. We’ll ask them to modify an existing google sheet so that, when updated, will return a specific value within the given cell.
```

Comment:
```suggestion
Their first assignment is mostly an exercise in following instructions. We’ll ask them to modify an existing Google Sheet so that, when updated, will return a specific value within the given cell.
```